client/snapper.cc

```diff
@@ -1318,12 +1318,12 @@ command_rollback(ProxySnappers* snappers, ProxySnapper* snapper)
 	exit(EXIT_FAILURE);
     }
 
-    pair<bool, unsigned int> previous_default = filesystem->getDefault();
+    ProxySnapshots& snapshots = snapper->getSnapshots();
 
-    if (previous_default.first && scd1.description == "rollback backup")
-	scd1.description += sformat(" of #%d", previous_default.second);
+    ProxySnapshots::iterator previous_default = snapshots.getDefault();
 
-    ProxySnapshots& snapshots = snapper->getSnapshots();
+    if (previous_default != snapshots.end() && scd1.description == "rollback backup")
```

Comment:
I'd just prefer some `#define` or a constant to have a single place for this string. Currently you need to update two places when changing it and that's quite fragile...